### ISSUE TITLE
Add section describing OCFL Object ZIP

### ISSUE THREAD

**Issue by zimeon** [OCFL/spec#181]
We currently talk about a zipped object only in the definition of [OCFL Object Root](https://ocfl.io/draft/spec/#dfn-ocfl-object-root). I think we need a new 3.x section on this (maybe 3.9)

**Comment by zimeon** [comment]
Yes, maybe 3.1.1 would be better. Message of that numbering is that the object structure is the same, expanded or zipped, but this is a packaging option

**Comment by ahankinson** [comment]
Editors Call Oct. 3 2018: Decision was to defer all reference to zipping in OCFL
